tusclient/uploader/baseuploader.py

```diff
@@ -119,6 +120,7 @@ def __init__(self, file_path: Optional[str] = None, file_stream: Optional[IO] =
         self.file_stream = file_stream
         self.stop_at = self.get_file_size()
         self.client = client
+        self.client_cert = client_cert
```

Comment:
Set to client.client_cert if client_cert is None? Probably makes sense. Not sure

Reply:
Could you explain what you mean? I don't fully understand the problem.

Reply:
I was just wondering about what should be done if this constructor is called with `client_cert=None` but the client already was constructed with a client certificate. But I think this should be considered incorrect use.

It's a bit complicated right now because baseuploader, uploader and TusRequest all do "HTTP" stuff and all need to use the client cert. But changing that would be more than I dare to touch atm. See: https://github.com/tus/tus-py-client/issues/77.

Reply:
Why does the BaseUploader class constructor needs its own `client_cert` argument at all? Couldn't it access it via `client.client_cert`? Then we don't have to deal with cases where `client_cert` on BaseUploader differs from `client_cert` on `Client`.

Reply:
Looking more at it, the reason seems to be that this way one can also set `client_cert` without providing a TusClient at all. However, I don't think that this is a use case we need to support. Just like `headers` can also be set through TusClient, we can also provide `client_cert` through a TusClient. Then we avoid having duplicate ways to achieve the same thing. Does that make sense?

Reply:
I've changed it so certificate is only set via TusClient. Makes the code cleaner.